Configuration/PyReleaseValidation/python/relval_upgrade.py

```diff
@@ -31,8 +31,11 @@ def makeStepName(key,frag,step,suffix):
             for step in upgradeProperties[year][key]['ScenToRun']:                    
                 stepMaker = makeStepName
                 if 'Sim' in step:
-                    if 'HLBeamSpot' in step and '14TeV' in frag:
-                        step = 'GenSimHLBeamSpot14'
+                    if 'HLBeamSpot' in step:
+                        if '14TeV' in frag:
+                            step = 'GenSimHLBeamSpot14'
+                        if 'CloseByParticle' in frag:
```

Comment:
Do you want to adjust only
https://github.com/cms-sw/cmssw/blob/master/Configuration/PyReleaseValidation/python/upgradeWorkflowComponents.py#L1386

or also
https://github.com/cms-sw/cmssw/blob/master/Configuration/PyReleaseValidation/python/upgradeWorkflowComponents.py#L1387-L1394
(as they are also clsebyparticle gun for HGCAL, https://github.com/cms-sw/cmssw/pull/27631)

Reply:
Yeah true, better to pattern match also CE_H and CE_E. Can I just update the code in my repository and it will redo checks?

Reply:
I will re-trigger the code testing. Feel free to push an updated one.

Reply:
Thanks! Pushed!